app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -91,7 +94,6 @@ export default function CompleteSignupForm() {
     register,
     handleSubmit,
     setValue,
-    getValues,
     errors,
   } = useForm<SignupInputs>({
     defaultValues: { city: "", location: {} },
```

Comment:
city is no longer in the form, right?

Reply:
display address is still called "city" in the user profiles

Reply:
Nah you're right, I reshuffled some more stuff